File: app/src/main/java/com/example/client/screens/reset/activity/IPasswordResetView.java
package com.example.client.screens.reset.activity;

import com.example.client.models.message.MessageModel;

public interface IPasswordResetView {
    void showViewPassword(MessageModel message);
    void resetPassword(MessageModel message);
    void sendEmail(MessageModel message);
    void showSendEmailLoading();
    void hideSendEmailLoading();
    void showVertifiLoading();
    void hideVertifiLoading();
    void showResetLoading();
    void hideResetLoading();
}

File: app/src/main/java/com/example/client/screens/reset/activity/IPasswordResetView.kt
package com.example.client.screens.reset.activity;

public interface IPasswordResetView {
    void showViewPassword();
    void onConfirmReset();
    void sendRequestComplete();
    void showSendEmailLoading();
    void hideSendEmailLoading();
    void showVerifyLoading();
    void hideVerifyLoading();
    void showResetLoading();
    void hideResetLoading();
    void showErrorMessage(int errMessage);
}
